test/e2e/antreapolicy_test.go

```diff
@@ -248,15 +248,24 @@ func testUpdateValidationInvalidACNP(t *testing.T) {
 	builder = builder.SetName("acnp-applied-to-update").
 		SetAppliedToGroup([]ACNPAppliedToSpec{{PodSelector: map[string]string{"pod": "a"}}}).
 		SetPriority(1.0)
-	builder.AddIngress(ProtocolTCP, nil, nil, nil, nil, nil, nil, nil, nil, map[string]string{"pod": "b"}, nil,
-		nil, nil, nil, nil, nil, nil, crdv1beta1.RuleActionAllow, "", "", nil)
+	builder.AddIngress(
+		IngressBuilder{
+			Protoc:      ProtocolTCP,
+			PodSelector: map[string]string{"pod": "b"},
+			Action:      crdv1beta1.RuleActionAllow,
+		})
```

Comment:
This format is more common and can reduce the number of lines and indentation.
```suggestion
	builder.AddIngress(IngressBuilder{
		Protoc:      ProtocolTCP,
		PodSelector: map[string]string{"pod": "b"},
		Action:      crdv1beta1.RuleActionAllow,
	})
```

Reply:
I've inline the usage of the builder in this PR (https://github.com/antrea-io/antrea/pull/7100). You can see the commit here: https://github.com/antrea-io/antrea/pull/7100/commits/42241ffdde0f7087ae7c4494cd3f217c99905102